scylla-cql/src/frame/response/result.rs

```diff
@@ -835,7 +853,7 @@ fn deser_result_metadata(
             .then(|| deser_table_spec(buf))
             .transpose()?;
 
-        deser_col_specs(buf, global_table_spec, col_count)?
+        deser_col_specs_owned(&mut *buf, global_table_spec, col_count)?
```

Comment:
nit: Why `buf` argument transformed to `&mut *buf`? I believe it could remain as `buf`. (`deser_result_metadata` function)

Reply:
This is a leftover.